kelvin/urls.py

```diff
@@ -36,7 +37,7 @@
     path("webpush/", include("webpush.urls")),
     # For django-tasks-scheduler
     path("scheduler/", include("scheduler.urls")),
-]
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Why was this change needed?

Reply:
Debug purposes, forgot to remove, removed